src/Nethermind/Nethermind.Facade/IBlockchainBridge.cs

```diff
@@ -28,9 +27,10 @@ public interface IBlockchainBridge : ILogFinder
         TxReceipt GetReceipt(Hash256 txHash);
         (TxReceipt? Receipt, TxGasInfo? GasInfo, int LogIndexStart) GetReceiptAndGasInfo(Hash256 txHash);
         (TxReceipt? Receipt, Transaction? Transaction, UInt256? baseFee) GetTransaction(Hash256 txHash, bool checkTxnPool = true);
-        CallOutput Call(BlockHeader header, Transaction tx, CancellationToken cancellationToken);
+        CallOutput Call(BlockHeader header, Transaction tx, Dictionary<Address, AccountOverride>? stateOverride, CancellationToken cancellationToken);
+        CallOutput Call(BlockHeader header, Transaction tx, CancellationToken cancellationToken) => Call(header, tx, null, cancellationToken);
```

Comment:
```suggestion
        CallOutput Call(BlockHeader header, Transaction tx, Dictionary<Address, AccountOverride>? stateOverride = null, CancellationToken cancellationToken = default);
```

Reply:
That was initially done to avoid changes in `BlockchainBridgeTests`, but it's probably a bit overdue, with 40+ changed files 😅